src/redux/reducers/experiments/samplesDelete.js

```diff
@@ -6,7 +6,7 @@ import initialState from './initialState';
 const samplesDelete = produce((draft, action) => {
   const { experimentId, sampleUuids } = action.payload;
 
-  _.pullAll(draft[experimentId].samples, sampleUuids);
+  _.pullAll(draft[experimentId].sampleIds, sampleUuids);
```

Comment:
redux stores the ids as `samples` in `projects` but as `sampleIds` in `experiments`